examples/instance_default/instance_default.tf

```diff
@@ -1,39 +1,60 @@
-// Copyright (c) 2018, Oracle and/or its affiliates. All rights reserved.
+// Copyright (c) 2018, 2021 Oracle and/or its affiliates. All rights reserved.
```

Comment:
"All rights reserved" is not needed anymore. Copyright notice should be as below:
// Copyright (c) 2018, 2021 Oracle and/or its affiliates.